### PR TITLE
PXC-3562 Release notes for 8.0.22-13.1

### PR DESCRIPTION
**Pull request by patrickbirch**
Modified:   source/conf.py
new file:   source/release-notes/Percona-XtraDB-Cluster-8.0.22-13.1
modified:   source/release-notes/release-notes_index
Merged with 5.7 version

### CODE REVIEW

File: doc/source/release-notes/Percona-XtraDB-Cluster-8.0.22-13.1.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+
+
+Known Issues (unfixed problems that you should be aware of)
+================================================================================
+
+* :jirabug:`PXC-3039`: No useful error messages if an SSL-disabled node tries to join an SSL-enabled cluster
+* :jirabug:`PXC-3092`: Log a warning at startup if a keyring is specified, but cluster traffic encryption is turned off
+* :jirabug:`PXC-3093`: Completed SST Transfer incorrectly logged by garbd (Timing is not correct)
+* :jirabug:`PXC-3159`: Modify the error handling to close the communication channels and abort the joiner node when the donor crashes
```
Comment (venkatesh-prasad-v) [inline doc/source/release-notes/Percona-XtraDB-Cluster-8.0.22-13.1.rst]: PXC-3159 was fixed in 8.0.20-11 and is still marked as a known issue here. I think we should remove it from this list.

**Review by venkatesh-prasad-v — APPROVED** [review]
Looks good to me

File: doc/source/release-notes/Percona-XtraDB-Cluster-8.0.22-13.1.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-XtraDB-Cluster-8.0.22-13.1.rst]: Removed when updated.

**Review by nastena1606 — APPROVED** [review]
LGTM

**Pull request merged**